portals/knowledge-management/src/portal.js

```diff
@@ -70,6 +70,11 @@ customElements.define(
                         name: 'view-category',
                         component: ViewCategory,
                     },
+                    {
+                        path: baseUrl + '/subcategories/:subCategorySlug',
```

Comment:
What do you think about this being related to the parent category. So `baseUrl + '/categories/:parentCategorySlug/:categorySlug'`.

This way may make more sense for the pathing